File: cop_mobile/cop_mobile/cop_mobile/Model/FCMToken.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace cop_mobile.Model
{
    public class FCMToken
    {
        public long fcmTokenId { get; set; }
        public Member member { get; set; }
        public string token { get; set; }
        public string deviceId { get; set; }

        public FCMToken(long fcmTokenId, Member member, string deviceId, string token)
        {
            this.fcmTokenId = fcmTokenId;
            this.member = member;
            this.deviceId = deviceId;
            this.token = token;
        }

        public FCMToken(Member member, string deviceId, string token)
        {
            this.member = member;
            this.deviceId = deviceId;
            this.token = token;
        }

        public FCMToken()
        {
        }
    }
}

File: web-server/src/main/java/com/churchofphilippi/webserver/model/FCMToken.java
package com.churchofphilippi.webserver.model;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import javax.persistence.*;

@Entity
@Table(name = "fcm_token")
@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class FCMToken {

    @Id
    @SequenceGenerator(
            name = "fcm_token_sequence",
            sequenceName = "fcm_token_sequence",
            allocationSize = 1
    )
    @GeneratedValue(
            strategy = GenerationType.SEQUENCE,
            generator = "fcm_token_sequence"
    )
    @Column(
            name = "fcm_token_id",
            updatable = false
    )
    private Long fcmTokenId;

    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "member_id")
    @JsonIgnoreProperties({"hibernateLazyInitializer", "handler"})
    private Member member;

    @Column(nullable = false)
    private String token;

    @Column(nullable = false)
    private String deviceId;
}
